packages/react-magma-dom/README.md

```diff
@@ -0,0 +1,316 @@
+# React magma
+
+## Known Issues
```

Comment:
Parts of the readme that were removed in #25 are back

Reply:
Just noticed this is the react-magma-dom readme... it gets clobbered by the top level one [on publish](https://github.com/cengage/react-magma/blob/d33dc7ec958bae0e8aca248ce2cf176c1bf11a51/package.json#L26)

We want the readme at the top level for GitHub ... but inside the published package for npm 🤔